vault/logical_system_paths.go

```diff
@@ -300,15 +301,33 @@ func (b *SystemBackend) configPaths() []*framework.Path {
 			Operations: map[logical.Operation]framework.OperationHandler{
 				logical.ReadOperation: &framework.PathOperation{
 					Callback: b.handleLoggersRead,
-					Summary:  "Read the log level for all existing loggers.",
+					Responses: map[int][]framework.Response{
+						http.StatusOK: {{
+							Description: "OK",
+							Fields:      map[string]*framework.FieldSchema{
+								// TODO need to understand how we can find out all loggers
+							},
+						}},
+					},
+					Summary: "Read the log level for all existing loggers.",
 				},
 				logical.UpdateOperation: &framework.PathOperation{
 					Callback: b.handleLoggersWrite,
-					Summary:  "Modify the log level for all existing loggers.",
+					Responses: map[int][]framework.Response{
+						http.StatusOK: {{
```

Comment:
I think most of the `"Update"` & `"Delete"` statuses should be 204 (`http.StatusNoContent`). If the response handler returns `nil, nil`, it will be translated into a 204 by the routing logic. The ones with data and/or warnings will be returned as 200s.

Reply:
done